src/connections/destinations/catalog/actions-singlestore/index.md

```diff
@@ -27,37 +27,62 @@ This destination is maintained by SingleStore. For any issues with the destinati
+
+**Note:**  
+- The Data API is enabled by default for all SingleStore Cloud workspaces.  
+- Segment always uses the Data API (typically on port 443).  
+- If you are using a self-hosted or development SingleStore deployment (such as the SingleStore-dev image), the Data API may run on a different port. Refer to your deployment’s documentation or settings to confirm the correct port.
+
+For more details, see the [SingleStore Data API documentation](https://docs.singlestore.com/cloud/reference/data-api/){:target="_blank"}.
 
 ## Database structure
-Segment creates a table called `segment_raw_data` and writes data to your SingleStore database using the following schema:
+Segment writes data to your specified table in SingleStore (by default, this is `segment_data`) using the following schema:
 
 | Column |	Type	| Description |
 | -------- | ------ |  ----------- |
-| `message` |	JSON (utf8_bin) |	The entire message received from Segment, in JSON format |
-| `timestamp` |	datetime |	The timestamp of when the event was generated |
-| `event` |	VARCHAR(255) |	The event name (for Track events) |
-| `messageId` |	VARCHAR(255) |	The unique identifier of the event to ensure there is no duplication |
-| `type` |	VARCHAR(255) |	The type of the event (for example, Identify, Track, Page, Group) |
-
-
-### Accessing nested data
-To query specific data from the Segment event within SingleStore, you can de-reference the JSON pointer within the message column. For example:
+| `messageId` |	TEXT |	A unique identifier for the event to ensure there is no duplication. |
+| `timestamp` |	Datetime(6) |	The timestamp of when the event was generated |
+| `type` |	TEXT |	The type of the event (e.g., "track", "identify", "page", "screen", "group", "alias"). |
+| `event` |	TEXT |	The name of the event. Only required for "track" events. |
+| `name` |	TEXT |	The name of the page or screen. |
+| `properties` | JSON |	The properties of the track, page or screen event. |
+| `userId` |	TEXT |	The user ID associated with the event. |
+| `anonymousId` |  TEXT |	The anonymous ID associated with the event. |
+| `groupId` |  TEXT |	The group ID associated with the event. |
+| `traits` |  JSON |	The traits of the user associated with the event. |
+| `context` |	JSON |	The context of the event. Contains user environment information. |
+
+
+### Accessing data
+To query specific data from the Segment event within SingleStore, you can select columns directly. For example, to retrieve the `properties` object:
 
 ```sql
-SELECT message::properties FROM segment_raw_data;
+SELECT properties FROM your_table_name;
 ```
 
-This query retrieves the properties object from the JSON message, allowing you to work with nested event data.
+This query retrieves the `properties` object from the specified table, allowing you to work with event data.
```

Comment:
I don't think that this section in the current format is needed.
I would rewrite it to show how to access nested information from the traits/context columns.